task_sdk/src/airflow/sdk/execution_time/comms.py

```diff
@@ -97,6 +107,10 @@ def from_conn_response(cls, connection_response: ConnectionResponse) -> Connecti
 class VariableResult(VariableResponse):
     type: Literal["VariableResult"] = "VariableResult"
 
+    @classmethod
+    def from_variable_response(cls, variable_response: VariableResponse) -> VariableResult:
+        return cls(**variable_response.model_dump())
```

Comment:
Can we add the docstring like xcoms for this and connection?

Reply:
Done in https://github.com/apache/airflow/pull/45104/commits/08d648c0c0209eec8573c44b350864f454ba7e1f